quotes/ContentView.swift

```diff
@@ -0,0 +1,53 @@
+//
+//  ContentView.swift
+//  quotes
+//
+//  Created by Jason Zurita on 4/05/21.
+//
+
+import SwiftUI
+
+struct ContentView: View {
+    @State var quote: Quote?
+    var body: some View {
+        VStack {
+            if let q = quote {
+                Spacer()
+                Spacer()
+                Text(q.content)
+                Text("— \(q.author)")
+                Spacer()
+                Button("New") {
+                    loadNewQuote()
+                }
+                Spacer()
+                Spacer()
+
+            } else {
+                Text("Loading...")
+            }
+        }
+        .onAppear {
+            loadNewQuote()
+        }
+    }
+
+    private func loadNewQuote() {
+        let url = URL(string: "https://api.quotable.io/random?tags=technology,famous-quotes")!
+
+        let task = URLSession.shared.dataTask(with: url) {(data, response, error) in
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/jasonzurita/swift-linting-formatting-workshop/blob/43daff16dec4f87c24ec43333644d1a74bf11384/quotes/ContentView.swift#L38"></span>Unused parameter "error" in a closure should be replaced with _.<br />
<code>unused_closure_parameter</code> <code>ContentView.swift:38</code></td>
    </tr>
  </tbody>
</table>